src/script/interpreter.cpp

```diff
@@ -1142,24 +1142,24 @@ class CTransactionSignatureSerializer {
 
 uint256 GetPrevoutHash(const CTransaction& txTo) {
     CHashWriter ss(SER_GETHASH, 0);
-    for (unsigned int n = 0; n < txTo.vin.size(); n++) {
-        ss << txTo.vin[n].prevout;
+    for (const auto& txIn : txTo.vin) {
```

Comment:
Hungarian notation uses the lowercase type identifier (eg, "tx") followed by camel case variable name (eg, "In"). While it's not required to use this notation, your "txIn" variable name *looks* like it  is, while not matching the usage expectation.